content/docs/command-reference/repro.md

```diff
@@ -14,7 +14,7 @@ usage: dvc repro [-h] [-q | -v] [-f] [-s] [-c <path>] [-m] [--dry] [-i]
                  [targets [targets ...]]
 
 positional arguments:
-  targets        Stage or .dvc file to reproduce
+  targets        Stage, path to dvc.yaml or .dvc file to reproduce
```

Comment:
Why comma? Just need to copy/paste the text from https://github.com/iterative/dvc.org/pull/1861

Reply:
Or use commas consistently (one is missing after "dvc.yaml" above).